Data/ByteString.hs

```diff
@@ -1690,6 +1694,10 @@ putStr :: ByteString -> IO ()
 putStr = hPut stdout
 
 -- | Write a ByteString to stdout, appending a newline byte
+--
+-- This is not atomic.
```

Comment:
It would be helpful to say what __is__ atomic, to provide a hint for users interested to achieve property. "Unlike `putStr`, this function is not atomic" or something along this.